cronenbroguelike/util.py

```diff
@@ -11,7 +11,7 @@ def read_overridable_config(default_file_name, override_file_name=None):
         override_file_name = re.sub(r"\.default(\.json)$", r"\1", default_file_name)
 
     def _load(file_name):
-        with open(file_name, "r") as inp:
+        with open(os.path.join("config", file_name), "r") as inp:
```

Comment:
Maybe make this an absolute path, like `os.path.join(os.dirname(__file__), 'config', file_name)` or summat

Reply:
nah

Reply:
:latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross: :latin_cross:  JESUS TAKE THE WHEEL

Reply:
> nah

:middle_finger: